package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/utils",
-  "version": "8.4.0",
+  "version": "8.5.0",
```

Comment:
Should this be 9.0.0?

Reply:
```suggestion
  "version": "9.0.0",
```

Reply:
Fixed! The yarn run will need to wait until superstruct 3.0.0 is released.